health/records.go

```diff
@@ -0,0 +1,76 @@
+package health
+
+import (
```

Comment:
Unit tests for memoryRecord.go and records.go?

Reply:
Added.